frontend/src/styles/main.css

```diff
@@ -0,0 +1,4 @@
+
+.user-select-text {
+    user-select: text;
+}
```

Comment:
I was supposing we're going to override standard Vuetify styles to make the labels selectable.

Reply:
Should this be done with all labels or just per page as needed?

Reply:
Globally, for all labels.